pages/cookies.js

```diff
@@ -56,7 +56,12 @@ const Cookies = () => (
         font-weight: 300; 
       }
 
-     @media (max-width: 720px) {
+      .reference {
```

Comment:
Shouldn't it just be ```a {```?

Reply:
Fixed in `78a66c3` commit.

As said yesterday, I'm interested by more information about the performances of a css class but imho it's not blocking for the bug fix in this PR.